Sources/MapboxCoreNavigation/EHorizon/ElectronicHorizonEdgeMetadata.swift

```diff
@@ -83,7 +83,8 @@ extension ElectronicHorizon.Edge {
             laneCount = native.laneCount as? UInt
             altitude = native.meanElevation as? Double
             curvature = UInt(native.curvature)
-            countryCode = native.countryCode
+            // TODO: Make sure to use correct country code.
+            countryCode = native.countryCodeIso3
```

Comment:
Apple platforms standardize on ISO 3166-1 alpha-2 codes, so ignore the alpha-3 code.

Reply:
MapboxNavigationNative doesn’t support alpha-2 codes as of v47.0.0, but it’ll come in the next release, so I made the documentation vague enough to allow for either for the time being. When we upgrade MapboxNavigationNative again, we’ll want to switch over to alpha-2 codes.

Reply:
Never mind, I had a stale build. The alpha-2 code is available as well.